### PR TITLE
Model.init.fix

### PR DESCRIPTION
**Pull request by 5yler**
### Background

- Fixes unrecoverable server error occurring if robot user stopped on startup and driver killed
- Publish u-stop status in loop before connection to robot established

### What's new
- ✅ [New feature description. Only a single new major feature is allowed per PR.]
- ❌ New feature is accompanied by new test: [name and description of new test].

### Tests
1. ❌ Tested on simulated robot: [Describe the tests/commands used.]
2. ✅ Tested on physical robot: tried restarting driver on u-stopped robot, verified successful connection

### Quality
3. ✅New code is written in pure C++17 to the best of my knowledge.
4. ✅ New code follows established C++17 best practices ([C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines)).
5. ✅ New code passed `clang-format` and `cpplint` at least, if not all of our static code analysers.
6. ❌ I have included Doxygen-style documentation in the new C++ code.

### Note to reviewers
Please verify that all sections are accurately filled out and that all 6 numbered entries above are present and ticked/checked off, with their requirements met, if applicable.

### CODE REVIEW

File: src/driver/franka_plan_runner.cc [hunk 1]
```diff
@@ -193,17 +190,13 @@ int FrankaPlanRunner::RunFranka() {
             fmt::format("robot cannot receive commands in mode: {} at startup",
                         utils::RobotModeToString(current_mode))};
         comm_interface_->PublishDriverStatus(false, err_msg);
-        if (t_now - t_last_main_loop_log_ >= std::chrono::seconds(1)) {
-          dexai::log()->error("RunFranka: {}", err_msg);
-          t_last_main_loop_log_ = t_now;
+        if (current_mode == franka::RobotMode::kUserStopped) {
+          comm_interface_->PublishBoolToChannel(
+              utils::get_current_utime(),
+              comm_interface_->GetUserStopChannelName(), true);
```
Comment (duanyutong) [inline src/driver/franka_plan_runner.cc:196]: move handling for `franka::RobotMode::kUserStopped` mode before L188 `else if`, as opposed to inside `else if (current_mode != franka::RobotMode::kIdle) {`

File: src/driver/franka_plan_runner.cc [hunk 2]
```diff
@@ -237,6 +230,9 @@ int FrankaPlanRunner::RunFranka() {
   status_ = RobotStatus::Running;  // init done, define robot as running
   bool status_has_changed {true};
 
+  // robot model for impedance control calculations
+  model_ = std::make_unique<franka::Model>(robot_->loadModel());
```
Comment (duanyutong) [inline src/driver/franka_plan_runner.cc:234]: this is the key change? cannot attempt to load model when ustopped? add comments

Comment (5yler) [reply src/driver/franka_plan_runner.cc:234]: Yes, this is the key change. Not sure if loading model was successful, but exiting the program once model was loaded and robot u-stopped caused an unrecoverable server error

File: src/driver/franka_plan_runner.cc (hunk 1 above)
Comment (5yler) [reply src/driver/franka_plan_runner.cc:196]: I don't see why this is necessary. This is simpler and cleaner

Comment (5yler) [reply src/driver/franka_plan_runner.cc:196]: Avoid duplicating time-checking logic and printout

File: src/driver/franka_plan_runner.cc (hunk 2 above)
Comment (5yler) [reply src/driver/franka_plan_runner.cc:234]: Still need to test and make sure it behaves OK if the robot is u-stopped AFTER init, and then the driver is killed

File: src/driver/franka_plan_runner.cc (hunk 1 above)
Comment (duanyutong) [reply src/driver/franka_plan_runner.cc:196]: that section is structured to handle known individual cases first in some order, and then handle everything `else`. you cannot put a special case under the last `else` which breaks the code structure

**Review by duanyutong — APPROVED** [review]
plat & style

**Pull request merged**